From f28130d1365f43e28bb5490ca821340a0b37d262 Mon Sep 17 00:00:00 2001
From: Wolfgang Bauer <wbauer@tmo.at>
Date: Mon, 3 Apr 2017 12:53:31 +0200
Subject: [PATCH] Fix error reported by GCC7 (boo#1031317)

/home/abuild/rpmbuild/BUILD/kde-workspace-4.11.22/libs/oxygen/oxygenhelper.cpp:
In member function 'bool Oxygen::Helper::hasHint(WId, Atom) const':
/home/abuild/rpmbuild/BUILD/kde-workspace-4.11.22/libs/oxygen/oxygenhelper.cpp:1092:21:
error: ISO C++ forbids comparison between pointer and integer
[-fpermissive]
          if( data == None || n != 1 ) return false;
                      ^~~~
---
 libs/oxygen/oxygenhelper.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: kde-workspace-4.11.22/libs/oxygen/oxygenhelper.cpp
===================================================================
--- kde-workspace-4.11.22.orig/libs/oxygen/oxygenhelper.cpp
+++ kde-workspace-4.11.22/libs/oxygen/oxygenhelper.cpp
@@ -1089,7 +1089,7 @@ namespace Oxygen
             &data);
 
         // finish if no data is found
-        if( data == None || n != 1 ) return false;
+        if( data == NULL || n != 1 ) return false;
         else return *data;
 
     }
Index: kde-workspace-4.11.22/kcontrol/randr/krandrtray.cpp
===================================================================
--- kde-workspace-4.11.22.orig/kcontrol/randr/krandrtray.cpp
+++ kde-workspace-4.11.22/kcontrol/randr/krandrtray.cpp
@@ -361,7 +361,7 @@ void KRandRSystemTray::populateMenu(KMen
 					if (screen->activeCount() != 1)
 					{
 						action = outputMenu->addAction(i18n("Disable"));
-						if (output->crtc() == None)
+						if (output->crtc() == NULL)
 						{
 							QFont font = action->font();
 							font.setBold(true);
